i3pystatus/network.py

```diff
@@ -156,6 +156,7 @@ def extract_wireless_info(self, interface):
             info["quality"] = quality["quality"]
         info["quality"] *= 100
         info["quality_bar"] = make_bar(info["quality"])
+        info["quality"] = '{:.0f}'.format(info["quality"])
```

Comment:
This breaks existing configurations with float formatting. better use `round` here.